tests/test_skip_remote_data.py

```diff
@@ -49,19 +55,37 @@ def test_skip_remote_data_astropy(pytestconfig):
     # astropy.test() has remote_data=none but we still got here somehow,
     # so fail with a helpful message
 
-    if pytestconfig.getoption('remote_data') == 'none':
-        pytest.fail('@remote_data was not skipped with remote_data=none')
+    if pytestconfig.getoption('remote_data') in ('none', 'github'):
+        pytest.fail('@remote_data was not skipped with remote_data=none'
+                    'or remote_data=github')
 
     # Test Astropy URL
     download_file(ASTROPY_DATA_URL + 'galactic_center/gc_2mass_k.fits')
 
     # Test non-Astropy URL
     if pytestconfig.getoption('remote_data') == 'astropy':
         with pytest.raises(Exception) as exc:
-            download_file('http://www.google.com')
+            download_file(EXTERNAL_URL)
+        assert "An attempt was made to connect to the internet" in str(exc.value)  # noqa
+    else:  # remote_data=any
+        download_file(EXTERNAL_URL)
+
+
+@pytest.mark.remote_data(source='github')
+def test_skip_remote_data_github(pytestconfig):
+    if pytestconfig.getoption('remote_data') == 'none':
+        pytest.fail('@remote_data was not skipped with remote_data=none')
+
+    # Test GitHub URL
+    download_file('http://astropy.github.io')
```

Comment:
This should have been `GITHUB_DATA_URL`. Either rebase reverted this or I overlooked.